tools/integration_tests/operations/copy_dir_test.go

```diff
@@ -219,7 +219,4 @@ func TestCopyDirectoryInNonEmptyDirectory(t *testing.T) {
 
 	destSrc := path.Join(destDir, SrcCopyDirectory)
 	checkIfCopiedDirectoryHasCorrectData(destSrc, t)
-
-	os.RemoveAll(srcDir)
```

Comment:
why are these steps removed completely.

Reply:
As we are cleaning the bucket before every test begins, we don't need these steps now.